Wrappers/Python/cil/optimisation/functions/MixedL21Norm.py

```diff
@@ -72,15 +72,24 @@ def _proximal_step_numpy(tmp, tau):
     '''
     # Note: we divide x by tau so the cases of tau both scalar and 
     # DataContainers run
-    tmp /= np.abs(tau, dtype=np.float32)
-    res = tmp - 1
+    try:
+        tmp = np.abs(tau, dtype=np.float32)
```

Comment:
Why do you cast to float32?

Reply:
I guess there's no strict need. It is a relic of when `tau` was supposed to be an ndarray or a `DataContainer`.